go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/docker/docs
 go 1.20
 
 require (
-	github.com/compose-spec/compose-spec v0.0.0-20230810081227-631f10f1aabc // indirect
+	github.com/compose-spec/compose-spec v0.0.0-20230824062516-848a47ad5d0e // indirect
 	github.com/distribution/distribution v2.8.2+incompatible // indirect
 	github.com/docker/buildx v0.11.2 // indirect
 	github.com/docker/cli v24.0.5+incompatible // indirect
 	github.com/docker/compose-cli v1.0.35 // indirect
 	github.com/docker/distribution v2.8.2+incompatible // indirect
 	github.com/docker/scout-cli v0.20.0 // indirect
-	github.com/moby/buildkit v0.12.1 // indirect
+	github.com/moby/buildkit v0.12.2 // indirect
```

Comment:
For buildkit we should use the default branch as ref like it was before: https://github.com/docker/docs/blob/4bbeb2093f9a97b00ced7e8929ce6e9d29a2430a/_config.yml#L208

Reply:
It resolves to v0.12.2 automatically when I use `@master` 😕

Reply:
ah right we have created this release yesterday, sounds good then

Reply:
hum actually no it should be `v0.12.1-0.20230824004934-4376f3861b05`

Reply:
Hmm sorry, I was wrong, it resolves correctly to 0.12.1